airflow/providers/amazon/aws/links/emr.py

```diff
@@ -66,3 +81,100 @@ def get_log_uri(
         return None
     log_uri = S3Hook.parse_s3_url(cluster_info["LogUri"])
     return "/".join(log_uri)
+
+
+class EmrServerlessLogsLink(BaseAwsLink):
+    """Helper class for constructing Amazon EMR Serverless Logs Link."""
+
+    name = "Spark Driver stdout"
+    key = "emr_serverless_logs"
+
+    def get_link(
+        self,
+        operator: BaseOperator,
+        *,
+        ti_key: TaskInstanceKey,
+    ) -> str:
+        """
+        Link to Amazon Web Services Console.
+
+        :param operator: airflow operator
+        :param ti_key: TaskInstance ID to return link for
+        :return: link to external system
+        """
+        conf = XCom.get_value(key=self.key, ti_key=ti_key)
+        if not conf:
+            return ""
+        conn_id = operator.aws_conn_id
+        hook = EmrServerlessHook(aws_conn_id=conn_id)
```

Comment:
Is it actually work with something rather then "aws_default" ?

Reply:
As far as I know, it should, right? If somebody provides a different connection to the operator. I suppose I could pass in the conn_id itself instead of trying to get it from the operator.

Reply:
Confirmed that yes it works with a different connection.